src/inmanta/server/config.py

```diff
@@ -44,13 +44,39 @@
 db_name = Option("database", "name", "inmanta", "The name of the database on the postgresql server", is_str)
 db_username = Option("database", "username", "postgres", "The username to access the database in the PostgreSQL server", is_str)
 db_password = Option("database", "password", None, "The password that belong to the database user", is_str)
+
 db_connection_pool_min_size = Option(
-    "database", "connection_pool_min_size", 10, "Number of connections the pool will be initialized with", is_int
+    "database",
+    "connection_pool_min_size",
+    10,
+    "[DEPRECATED USE :inmanta.config:option:`database.service_connection_pool_min_size`] Number of connections the database connection pool will be initialized with",
```

Comment:
There is a way to indicate deprecation that neatly links in the docs

Reply:
Added `predecessor` to the new server options